cmd/bblfsh-sdk-tools/cmd/fixtures.go

```diff
@@ -9,10 +9,14 @@ import (
 
 	"google.golang.org/grpc"
 	protocol1 "gopkg.in/bblfsh/sdk.v1/protocol"
+	protocol2 "gopkg.in/bblfsh/sdk.v2/protocol"
+
+	"gopkg.in/bblfsh/sdk.v2/uast/nodes"
+	"gopkg.in/bblfsh/sdk.v2/uast/yaml"
 )
 
 const FixturesCommandDescription = "" +
-	"Generate integration tests' '.native' and '.uast' fixtures from source files"
+	"Generate integration tests' '.native', '.sem.uast', '.legacy' and '.proto' fixtures from source files"
 
 type FixturesCommand struct {
```

Comment:
In general, I don't see the reason to support this tool, since we already have a `bblfsh-cli` that can dump those files, and `fixture_test` will automatically generate missing fixture files. But if you have some use case in mind - let's do it.

Reply:
Motivation was similar to test plan above https://github.com/bblfsh/sdk/pull/314#issuecomment-429742318:
 1. fixtures in .proto is used for generating test mocks for some clients (applications)
 2. main use case for me is 
  ```
   $ bblfsh-sdk fixture test.py
   $ bblfsh-sdk ast2gv test.py.native
  ```

If there is a better way that I have missed for 2, please let me know